src/util.ts

```diff
@@ -51,10 +51,15 @@ export function getPrettyTime(ns: number): string {
 
 export function replaceWorkspaceFolderPathVariable(path: string): string {
   if (vscode.workspace.workspaceFolders !== undefined) {
-    path = path.replace("${workspaceFolder}", vscode.workspace.workspaceFolders![0].uri.toString());
-    path = path.replace("${workspacePath}", vscode.workspace.workspaceFolders![0].uri.path);
+    // here, uri.fsPath is used as this returns a usable path on both Windows and Unix
+    const workspaceFolderPath = vscode.workspace.workspaceFolders![0].uri.fsPath;
+    path = path.replace("${workspaceFolder}", workspaceFolderPath);
+    path = path.replace("${workspacePath}", workspaceFolderPath);
```

Comment:
So a `file://` URI is never what we want returned when using `workspaceFolder`? 

I know it's certainly not for `workspacePath`, but I was thinking the folder version expected the URI form.

Reply:
After testing, it turns out stripping out the `file://` prefix worked in all places where this was used.